### PR TITLE
fix bug - filters use in multiple widgets - using deepcopy

### PR DESCRIPTION
**Pull request by atalyaalon**
@ziv17 FYI - a bug I found
Did I miss any filter that wasn't copied?

### CODE REVIEW

File: anyway/infographics_utils.py [hunk 1]
```diff
@@ -1404,9 +1407,10 @@ def get_query(table_obj, filters, start_time, end_time):
 
 
 def get_accidents_stats(
-    table_obj, filters=None, group_by=None, count=None, start_time=None, end_time=None
+    table_obj, filters={}, group_by=None, count=None, start_time=None, end_time=None
```
Comment (zharpaz) [inline anyway/infographics_utils.py:1410]: Are you sure filters will never by None? If yes, then it is worthwhile to set its type, so lint will catch an issue.

**Comment by zharpaz** [comment]
> 
> 
> @ziv17 FYI - a bug I found
> Did I miss any filter that wasn't copied?

what about
AccidentCountByAccidentYearWidget

**Review by zharpaz — APPROVED** [review]
There are a few comments below.

**Comment by atalyaalon** [comment]
@ziv17 you are absolutely right, it's much more elegant to perform this in the constructor
I opened https://github.com/hasadna/anyway/pull/1749, closing this one

**Pull request closed without merging**